Utilities/bootstrap

```diff
@@ -321,10 +323,10 @@ def parse_manifest():
     # We have a *very* strict format for our manifest to make parsing more
     # robust.  We use this to determine the target and product definitions.
     target_pattern = re.compile(
-        r'Target\(.*?name: "(.*?)",\n *dependencies: (\[.*?\])\)',
+        r'Target\(.*?name: "(.*?)",\r?\n? *dependencies: (\[.*?\])\)',
         re.DOTALL | re.MULTILINE)
     product_pattern = re.compile(
-        r'Library\([\n ]*name: \"(.*?)\",[\n ]*type: (.*?),[\n ]*targets: (\[.*?\])[\n ]*\)',
+        r'Library\([\r\n ]*name: \"(.*?)\",[\r\n ]*type: (.*?),[\r\n ]*targets: (\[.*?\])[\r\n ]*\)',
         re.DOTALL | re.MULTILINE)
```

Comment:
I don't think we should need this, there should be a way to open the manifest file such that Python will automatically do the line ending conversion on reading the data.